### PR TITLE
add composer tag

### PR DESCRIPTION
**Pull request by Grummfy**
- composer tag
- author name
- other improvements

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
+    "keywords": ["TDD","atoum","test","unit testing","atoum-edition"],
+    "homepage": "http://www.atoum.org",
     "require": {
         "atoum/atoum": "^2.9",
         "atoum/stubs": "^2.5",
         "atoum/visibility-extension": "dev-self_register as 1.1",
         "atoum/config-extension": "dev-merge_prs as 1.0",
         "atoum/reports-extension": "dev-self_register as 2.9"
     },
+    "authors": [
```
Comment (agallou) [inline composer.json]: does adding authors will limit contributions ? plus this a a team repo, it's not only mine. should we keep that ? (also, if we keep it, the real name si prefered in the composer.json : https://getcomposer.org/doc/04-schema.md#authors )

**Comment by Grummfy** [comment]
here we are. I have removed the author tag

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {
     "name": "atoum/standard-edition",
+    "type": "package",
```
Comment (agallou) [inline composer.json]: there is no reference of a "package" type on https://getcomposer.org/doc/04-schema.md#type 
were dit you found it ?

Comment (Grummfy) [reply composer.json]: it could be anything

Comment (agallou) [reply composer.json]: "Only use a custom type if you need custom logic during installation" : there is no custom logic, maybe we should remove this attribute.

File: composer.json [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
+    "keywords": ["TDD","atoum","test","unit testing","atoum-edition"],
```
Comment (jubianchi) [inline composer.json]: please, could you add a space after each comma :)

**Review by jubianchi — CHANGES_REQUESTED** [review]

**Review by jubianchi — APPROVED** [review]

**Pull request merged**